pkg/pillar/evetpm/testhelper.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package evetpm
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/google/go-tpm/legacy/tpm2"
+)
+
+// SimTpmPath is the path to the SWTPM socket
+const SimTpmPath = "/tmp/eve-tpm/srv.sock"
+
+// SimTpmWaitForTpmReadyState waits for the SWTPM to be ready
+func SimTpmWaitForTpmReadyState() error {
+	for i := 0; i < 10; i++ {
+		rw, err := tpm2.OpenTPM(SimTpmPath)
+		if err != nil {
+			return fmt.Errorf("failed to open TPM: %w", err)
+		}
+
+		_, _, err = tpm2.GetCapability(rw, tpm2.CapabilityHandles, 1, uint32(tpm2.HandleTypeTransient)<<24)
```

Comment:
I'm not sure, but can we just use `TransientFirst` instead of `int32(tpm2.HandleTypeTransient)<<24`?

https://github.com/lf-edge/eve/blob/68f2f3da3f28bb5eac8fcfce76c59e3ddc89fb10/pkg/pillar/vendor/github.com/google/go-tpm/legacy/tpm2/constants.go#L351

Reply:
This is just a TPM call to get passed the RCRetry error, it doesn't really matter.